misc/services/mobile-webui/mobile-webui-frontend/src/components/ButtonQuantityProp.jsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import counterpart from 'counterpart';
 import PropTypes from 'prop-types';
 
-const ButtonQuantityProp = ({ qtyToPick, qtyPicked, uom }) => (
+const ButtonQuantityProp = ({ qtyTarget, qtyPicked, uom, appId }) => (
   <div className="row is-full is-size-7">
     <div className="picking-row-info">
-      <div className="picking-to-pick">{counterpart.translate('activities.picking.toPick')}:</div>
+      <div className="picking-to-pick">{counterpart.translate(`activities.${appId}.target`)}:</div>
       <div className="picking-row-qty">
-        {qtyToPick} {uom}
+        {qtyTarget} {uom}
```

Comment:
why we want to change that ? shouldn't remain consistent `qtyToPick` throughout the app ? I mean this is how we have it from the BE as well. I believe we should remain to qtyToPick